scripts/diagnose.sh

```diff
@@ -5,7 +5,7 @@ set +x
 
 print_usage() {
   echo "Usage:"
-  echo "    ./diagnose-juicefs.sh COMMAND [OPTIONS]"
+  echo "    ./$0 COMMAND [OPTIONS]"
```

Comment:
`./$0` will result in the following:

```
Usage:
    ././diagnose.sh COMMAND [OPTIONS]
COMMAND:
    help
        Display this help message.
    collect
        Collect pods logs of juicefs.
```

Maybe you can update to `$0`

Reply:
oh, sorry! it's updated!